### PR TITLE
feat: added Leptos Web Framework to Web Development

### PR DESCRIPTION
**Pull request by pixel365**

### CODE REVIEW

**Review by arikchakma — APPROVED** [review]

File: src/data/roadmaps/rust/content/108-web-development/106-leptos.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Leptos
+
+Leptos is a full-stack, isomorphic Rust web framework leveraging fine-grained reactivity to build declarative user interfaces.
+
+Learn more from the following resources:
+
+- [@official@Website](https://leptos.dev/)
```
Comment (offensive-vk) [inline src/data/roadmaps/rust/content/108-web-development/106-leptos.md:7]:
```suggestion
- [@official@Leptos Website](https://leptos.dev/)
```

File: src/data/roadmaps/rust/content/108-web-development/106-leptos.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- [@official@Leptos Documentation](https://book.leptos.dev/)
+- [@opensource@Leptos](https://github.com/leptos-rs/leptos)
+- [@video@YouTube](https://www.youtube.com/watch?v=aQOFJQ2JkvQ&list=PLg4xWRFolJIo0lZawIlEaYr180e1w-ZUH)
```
Comment (offensive-vk) [inline src/data/roadmaps/rust/content/108-web-development/106-leptos.md:10]:
```suggestion
- [@video@Learning Leptos](https://youtu.be/aQOFJQ2JkvQ?si=fFQeqh7Y5TS5HDcA)
```

**Review by offensive-vk — COMMENTED** [review]
Left some refractors.

**Comment by arikchakma** [comment]
> Left some refractors.

I have already merged it. Thank you!!

**Comment by offensive-vk** [comment]
bruh 😅

**Pull request merged**